<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1269</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="LieuxButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>LIEUX CULTURELS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restaurantsButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>480</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>RESTAURANTS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zonesButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>480</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ZONES</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="villageJOButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>490</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>VILLAGE JO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UsButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>490</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ABOUT US</string>
    </property>
   </widget>
   <widget class="QLabel" name="mapLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mapLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="emojiLabel1">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>250</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Paris 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="emojiLabel2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Paris 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="emojiLabel4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Paris 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="emojiLabel3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>170</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Paris 3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchlineEdit">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>0</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>0</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1269</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
